Makefile

```diff
@@ -20,9 +20,10 @@ test: generate
 	echo 'mode: atomic' > coverage.txt && go test -covermode=atomic -coverprofile=coverage.txt -v -race -timeout=30s ./...
 
 # Run the integration tests. Requires github token for scorecard (GITHUB_AUTH_TOKEN=<your token>)
+# To run it locally you can run the following command: make start-integration-service
 .PHONY: integration-test
 integration-test: generate check-env
-	go test -tags=integration ./...
+	ENT_TEST_DATABASE_URL='postgresql://guac:guac@localhost/guac?sslmode=disable' go test -tags=integration ./...
```

Comment:
I think if we'd like to do something like that, the command should handle the spinning up and tearing down of the services themselves? It would be good to have all that encoded in so the user doesn't have to worry about it.